Validation/GlobalDigis/python/globaldigis_analyze_cfi.py

```diff
@@ -31,5 +31,10 @@
     HCalDigi = cms.InputTag("simHcalDigis")
 )
 
-
-
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(globaldigisanalyze,
+    MuCSCStripSrc = "mixData:MuonCSCStripDigisDM",
+    MuCSCWireSrc = "mixData:MuonCSCWireDigisDM",
+    MuDTSrc = "mixData",
+    MuRPCSrc = "mixData",
```

Comment:
@makortel , may be I am wrong but is it OK to have comma in line 39?

Reply:
Indeed, I typically recommend using the trailing comma because it makes git diffs cleaner when new lines are added in a list.